# Ukrainian translation for xen
# Copyright (c) 2011 Rosetta Contributors and Canonical Ltd 2011
# This file is distributed under the same license as the xen package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2011.
#
msgid ""
msgstr ""
"Project-Id-Version: xen\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2008-03-31 17:40+0100\n"
"PO-Revision-Date: 2011-12-17 18:30+0000\n"
"Last-Translator: Yuri Chornoivan <yurchor@gmail.com>\n"
"Language-Team: Ukrainian <uk@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2024-03-25 09:38+0000\n"
"X-Generator: Launchpad (build aec24aef7a9042c99ef3e238d8b0ca01df9e1a9f)\n"

#: xen/xm/XenAPI.py:58
#, python-format
msgid "Internal error: %(1)s."
msgstr "Внутрішня помилка: %(1)s."

#: xen/xm/XenAPI.py:59
#, python-format
msgid "This map already contains %(1)s -> %(2)s."
msgstr "Ця карта вже містить %(1)s -> %(2)s."

#: xen/xm/XenAPI.py:60
#, python-format
msgid "The method %(1)s is unsupported."
msgstr "Метод %(1)s не підтримується."

#: xen/xm/XenAPI.py:61
#, python-format
msgid "The method %(1)s takes %(2)s argument(s) (%(3)s given)."
msgstr "Метод %(1)s приймає %(2)s аргумент(и/ів) (%(3)s передано)."

#: xen/xm/XenAPI.py:62
msgid "Permission denied."
msgstr "Доступ заборонено."

#: xen/xm/XenAPI.py:63
#, python-format
msgid ""
"Value \"%(2)s\" for %(1)s is not supported by this server.  The server said "
"\"%(3)s\"."
msgstr ""
"Значення \"%(2)s\" для %(1)s не підтримується цим сервером. Сервер "
"повідомив: \"%(3)s\"."

#: xen/xm/XenAPI.py:64
#, python-format
msgid "The %(1)s handle %(2)s is invalid."
msgstr "%(1)s дескриптор %(2)s помилковий."

#: xen/xm/XenAPI.py:65
msgid "You attempted an operation that was not allowed."
msgstr "Ви намагаєтесь здійснити неприпустиму операцію."

#: xen/xm/XenAPI.py:66
msgid ""
"The network you specified already has a PIF attached to it, and so another "
"one may not be attached."
msgstr ""
"Указана Вами мережа вже має під'єднаний PIF, тому інший не може бути "
"під'єднаний."

#: xen/xm/XenAPI.py:67
#, python-format
msgid "%(2)s"
msgstr "%(2)s"
